### PR TITLE
4.11 doc additions

### PR DESCRIPTION
**Pull request by aevesdocker**
⚠️ Only publish on the 28th July

What this changes:
- Adds info about the dark/light mode settings
- Adds info about the ability for the Extension marketplace to be sorted and updates the screenshot
- VDI doc
    - removes outdated content from the troubleshoot and faq page about nested virtualization
    - amends the title and URL
    - add a note about support 
    - add recommended configuration for VMWare
    - made it clearer this is for Windows
- add --user flag to mac install page
- amend privileged helper page

### CODE REVIEW

File: desktop/faqs/general.md [hunk 1]
```diff
@@ -106,11 +106,7 @@ serial port) to a container as it requires support at the hypervisor level.
 
 ### Can I run Docker Desktop in nested virtualization scenarios?
 
-Docker Desktop can run inside a Windows 10 VM running on apps like Parallels or
-VMware Fusion on a Mac provided that the VM is properly configured. However,
-problems and intermittent failures may still occur due to the way these apps
-virtualize the hardware. For these reasons, **Docker Desktop is not supported in
-nested virtualization scenarios**. It might work in some cases and not in others.
+In general, Docker recommends running Docker Desktop natively on either Mac, Linux, or Windows. However, Docker Desktop for Windows can run inside a virtual desktop provided the virtual desktop is properly configured. For more information, see [Enable nested virtualization](../vm-vdi.md)
```
Comment (MihaelaStoica) [inline desktop/faqs/general.md:109]: Replace `Enable nested virtualization` with `Running Docker Desktop in a VM or VDI environment`

File: desktop/mac/permission-requirements.md [hunk 2]
```diff
@@ -1,28 +1,29 @@
...
+- Sets up symlinks for the user
+- Ensures that `localhost` and `kubernetes.docker.internal` are present in `/etc/hosts`
 
-This approach will have certain limitations:
-- Docker Desktop would only be able to be run by one user account per machine, namely the one specified in the `-–user` flag.
-- Ports 1-79 would be blocked - the containers would run but the port won’t be exposed on the host.
-- Spindump diagnostics for fine grained CPU utilization would not be gathered.
+This approach has the following limitations:
+- Docker Desktop can only be run by one user account per machine, namely the one specified in the `-–user` flag.
+- Ports 1-79 are blocked. the containers will run but the port won’t be exposed on the host.
```
Comment (MihaelaStoica) [inline desktop/mac/permission-requirements.md:25]: Capital t in "the containers will run..."

Comment (aevesdocker) [reply desktop/mac/permission-requirements.md:25]: Thanks for catching that @MihaelaStoica

**Comment by aevesdocker** [comment]
@docker/docs-team Please can I get a review so it's ready for release with 4.11 tomorrow?

File: desktop/mac/permission-requirements.md [hunk 3]
```diff
@@ -1,28 +1,29 @@
...
+- /desktop/mac/privileged-helper/
 ---
 
 This page contains information about the permission requirements for running and installing Docker Desktop on Mac, the functionality of the privileged helper process `com.docker.vmnetd` and the reasoning behind this approach, as well as clarification about running containers as `root` as opposed to having `root` access on the host.
 
 ## Permission requirements
 
-While Docker Desktop does not generally require the user running or installing it to have `root` privileges, in the default use case it needs `root` access to be granted on the first run. The first time Docker Desktop is launched the user gets an admin prompt to grant permissions for a privileged helper service to be installed. For subsequent runs, no `root` privileges are required. 
+In the default set up flow, Docker Desktop for Mac does not require root privileges for installation but does require `root` access to be granted on the first run. The first time that Docker Desktop is launched the user gets an admin prompt to grant permissions for a privileged helper service to be installed. For subsequent runs, no `root` privileges are required.
```
Comment (jerae-duffin) [inline desktop/mac/permission-requirements.md:14]: Minor suggestion: "The first time that Docker Desktop is launched the user gets an admin prompt to grant permissions to install a privileged helper service."

File: desktop/mac/permission-requirements.md [hunk 4]
```diff
@@ -1,28 +1,29 @@
...
 
-From version 4.11, it will be possible to avoid running a privileged service in the background by using `com.docker.vmnetd` for setup during installation and disabling it at runtime. In this case the user will not be prompted on the first run. Administrators would be able to do that by using the `–-user` flag on the [install command](../install/mac-install.md#install-from-the-command-line) which would:
-- Uninstall the previous `com.docker.vmnetd` if present
-- Set up symlinks for the user
-- Ensure that `localhost` and `kubernetes.docker.internal` are present in `/etc/hosts`
+From version 4.11 of Docker Desktop for Mac it will be possible to avoid running the privileged helper service in the background by using the `--user` flag on the [install command](../install/mac-install.md#install-from-the-command-line). This will result in `com.docker.vmnetd` being used for set up during installation and then disabled at runtime. In this case, the user will not be prompted to grant root privileges on the first run of Docker Desktop. Specifically, the `--user` flag:
```
Comment (jerae-duffin) [inline desktop/mac/permission-requirements.md:18]: In version 4.11...it is possible

File: desktop/vm-vdi.md [hunk 5]
```diff
@@ -27,15 +24,16 @@ For troubleshooting problems and intermittent failures that are outside of Docke
 
 You must enable nested virtualization before you install Docker Desktop on a virtual machine.
 
-### Enable nested virtualization on VMware ESXi 
+### Enable nested virtualization on VMware ESXi
 
-Nested virtualization of other hypervisors like Hyper-V inside a vSphere VM [is not a supported scenario](https://kb.vmware.com/s/article/2009916). However, running Hyper-V VM in a VMware ESXi VM is technically possible and, depending on the version, ESXi includes hardware-assisted virtualization as a supported feature. 
+Nested virtualization of other hypervisors like Hyper-V inside a vSphere VM [is not a supported scenario](https://kb.vmware.com/s/article/2009916). However, running Hyper-V VM in a VMware ESXi VM is technically possible and, depending on the version, ESXi includes hardware-assisted virtualization as a supported feature. For internal testing, we used a VM that had 1 CPU with 4 cores and 12GB of memory.
```
Comment (jerae-duffin) [inline desktop/vm-vdi.md:29]: ,and

File: desktop/extensions.md [hunk 6]
```diff
@@ -86,8 +87,7 @@ Feedback can be given to an extension author through a dedicated Slack channel o
 extension, select:
     - Support
     - Slack
-    - Issues
-    You'll be sent to a page outside of Docker Desktop to submit your feedback.
+    - Issues. You'll be sent to a page outside of Docker Desktop to submit your feedback.
 
 If an extension does not provide a way for you to give feedback, contact us and we'll pass on the feedback for you.
```
Comment (jerae-duffin) [inline desktop/extensions.md:92]: How can they contact us? Is there a link to take them to a contact form?

Comment (aevesdocker) [reply desktop/extensions.md:92]: Yes there is a link in DD. But good point - I have made it clearer in the doc!

**Review by dockertopia — APPROVED** [review]
LGTM

**Pull request merged**